platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -461,6 +475,7 @@ private void voteInAllElections(@NonNull final EventImpl votingWitness) {
                         candidateWitness,
                         "coin-" + (countingVote.isSupermajority() ? "counting" : "sig"),
                         diff);
+                markerFileWriter.writeMarkerFile(COIN_ROUND_MARKER_FILE);
```

Comment:
Maybe this should log an error, to fail `jrs` tests

Reply:
added rate limited logger for coin rounds

Reply:
the `logVote` method does log an info message.  So I don't know if a coin round is technically an error.